### PR TITLE
URL wordwise selection fix

### PR DESCRIPTION
**Pull request by Grendel-Grendel-Grendel**
A quick fix that resolves issue #3 partially. I was unsure exactly how `getCursorColor` was getting its value from the config so I didn't touch that, though I added an exception list value to the relevant datatype and default.

I found that I had to add `.:/` which was three characters in order to successfully select the entire URL. If this is something unique to me perhaps there is something else weird going on.

### CODE REVIEW

File: src/Termonad/Config.hs [hunk 1]
```diff
@@ -62,4 +64,5 @@ defaultTMConfig =
     , cursorColor = lightgrey
     , scrollbackLen = 10000
     , confirmExit = True
+    , wordCharExceptions = ":./"
```
Comment (cdepillabout) [inline src/Termonad/Config.hs]: Could you change this to be something like the following:

`"-#%&+,./=?@\\_~\183:"`

This should be the default list of wordchar exceptions, plus `:`.

(As an aside, I was able to figure out the default list of wordchar exceptions by checking out what the output of [terminalGetWordCharExceptions](http://hackage.haskell.org/package/gi-vte-2.91.19/docs/GI-Vte-Objects-Terminal.html#g:40) is.)

That way, everything that is currently highlightable will continue to be highlightable, plus URLs.

Without this, things like email addresses `user@gmail.com` are no longer highlightable in one double-click.

File: src/Termonad/Term.hs [hunk 2]
```diff
@@ -277,6 +278,7 @@ createTerm handleKeyPress mvarTMState = do
   maybeCurrDir <- maybe (pure Nothing) cwdOfPid maybeCurrFocusedTabPid
   vteTerm <- terminalNew
   terminalSetFont vteTerm (Just tmStateFontDesc)
+  terminalSetWordCharExceptions vteTerm ":./"
```
Comment (cdepillabout) [inline src/Termonad/Term.hs]: I guess you should actually use `wordCharExceptions` here!

**Comment by Grendel-Grendel-Grendel** [comment]
Should be fixed in accordance with your comments.

**Comment by cdepillabout** [comment]
Thanks a lot!

**Pull request merged**